legion/engine/llri-dx/detail/queue.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file queue.cpp
+ * @copyright 2021-2021 Leon Brands. All rights served.
+ * @license: https://github.com/Legion-Engine/Legion-LLRI/blob/main/LICENSE
+ */
+
+#include <llri/llri.hpp>
+#include <llri-dx/directx.hpp>
+
+namespace LLRI_NAMESPACE
+{
+    result Queue::impl_submit(const submit_desc& desc)
+    {
+        HRESULT r;
+        unsigned long index;
+
+        if (desc.nodeMask != 0)
+            _BitScanForward64(&index, desc.nodeMask);
```

Comment:
this is msvc specific, but i don't know if it still works with clang on windows? Maybe try compiling it with clang on windows. if it compiles then it's all fine.

Reply:
I believe I tested this on ClangCL but I can check later today

Reply:
Tested on both MSVC and ClangCL and all unit tests pass